docker-compose.yml.dist

```diff
@@ -1,13 +0,0 @@
-version: '2'
-
-services:
-  payum-server:
-    image: payum/payum-server
-    environment:
```

Comment:
the file should be kept in repot

Reply:
But why? Imho much better to have docker-compose.yml which would get environment params from .env